primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -2228,6 +2239,16 @@
             }
         },
 
+        updateResponsiveness: function() {
+            if (this.options.autoDetectDisplay && !this.options.inline) {
+                var currentUI = this.options.touchUI;
+                this.options.touchUI = PrimeFaces.env.mobile || window.screen.width < 576;
```

Comment:
Shouldn't we move `window.screen.width < 576` to a utils function?

Reply:
.. and that value should be overridable (as with the first day of the week in the date picker)